src/app/core/entity/schema/entity-schema-field.ts

```diff
@@ -78,6 +78,8 @@ export interface EntitySchemaField {
    */
   entityReferenceRole?: EntityReferenceRole;
 
+  // TODO: possibly remove viewComponent + editComponent from schema and only infer via dataType (or overwrite in FormFieldConfig)
```

Comment:
For self-configurability it would definitely be helpful.

Reply:
noted for a possible implementation within the setup wizard, when we rework the datatype vs. component selection question.